src/styles/main.scss

```diff
@@ -22,3 +22,39 @@ body {
   align-items: center;
   justify-content: center;
 }
+
+.success {
+  position: absolute;
+  font-size: 20px;
+  margin: 10px;
+  background-color: green;
+  text-align: center;
+  border-radius: 15px;
+  padding: 30px;
+  right: 50px;
+
+  &__1 {
+    top: 50px;
+    right: 50px;
+  }
+
+  &__2 {
+    right: 50px;
+  }
+
+  &__3 {
+    bottom: 50px;
+    right: 50px;
+  }
+}
```

Comment:
Look how many times you use 50px. It would be better to create a variable for it